### PR TITLE
Update the threat model after review

### LINKED ISSUE #32: Fix minor issues from threat model feedback
A few small issues have been uncovered by the review of the threat model and these should be fixed, most easily together:

* the first asset should be `Authentication Token`, not `Application Identity`.
* add `Key Info Store` as an asset
* A-3 is given as a mitigation for Attacker "OS Signal"
* A-2 is given as a mitigation for Attacker "Hardware Result"
* show more clearly on the data flow diagram where the data assets might be exposed

### PR DESCRIPTION
**Pull request by hug-dev**
Fix #32

Signed-off-by: Hugues de Valon <hugues.devalon@arm.com>

### CODE REVIEW

**Comment by hug-dev** [comment]
[Rendered version](https://github.com/parallaxsecond/parsec-book/blob/157308b05a70cd5f112c5ee36f5cfda06e4cb98e/src/threat_model/threat_model.md)

File: src/threat_model/threat_model.md [hunk 1]
```diff
@@ -96,6 +95,30 @@ General asset to describe the fact that each client's request should be done in
...
+
+**Integrity** : Parsec expects this data to be valid.
+
+**Availability** : the data should be available when Parsec needs the specific key.
+
+### Logs- AS8
+
+General asset to describe the fact that each client's request should be done in a sane amount of
+time. The service should be available at any given point from the moment it is started by the
+administrator.
```
Comment (ionut-arm) [inline src/threat_model/threat_model.md:112]: Copypasta from `AS6`?

File: src/threat_model/threat_model.md [hunk 2]
```diff
@@ -133,6 +156,8 @@ This attacker uses the existing Listener endpoint, created by the service, to co
 This attacker uses the existing Listener endpoint, created by the service, to communicate with the
 client. It can also create a spoofed endpoint, mimicking the service's one.
 
+For protection of the clients' assets, please check the Parsec Clients Threat Models.
```
Comment (ionut-arm) [inline src/threat_model/threat_model.md:159]: 😆 check the what?

File: src/threat_model/threat_model.md (hunk 1 above)
Comment (hug-dev) [reply src/threat_model/threat_model.md:112]: Oops 🤭

File: src/threat_model/threat_model.md (hunk 2 above)
Comment (hug-dev) [reply src/threat_model/threat_model.md:159]: Well we can put the links once we have them :D

**Pull request merged**